cgfs.py

```diff
@@ -265,19 +387,23 @@ def get_file(self, path, start=None, expect_type=None):
             if part == '':  # pragma: no cover
                 continue
 
-            if not isinstance(file, Directory):  # pragma: no cover
-                raise FuseOSError(ENOTDIR)
-
-            if len(file.children) == 0:
-                if file.type == DirTypes.ASSIGNMENT:
-                    self.load_submissions(file)
-                elif file.type == DirTypes.SUBMISSION:
-                    self.load_submission_files(file)
-
-            file = file.get(part)
+            try:
+                if not file.children:
+                    if file.type == DirTypes.ASSIGNMENT:
+                        self.load_submissions(file)
+                    elif file.type == DirTypes.SUBMISSION:
+                        self.load_submission_files(file)
+            except AttributeError:  # pragma: no cover
+                if not isinstance(file, Directory):
+                    raise FuseOSError(ENOTDIR)
+                raise
```

Comment:
Shouldn't a specific error be raised here?

Reply:
This raises the old exception